API_README.md

```diff
@@ -146,6 +146,40 @@ Response:
 }
 ```
 
+Here is another example for registering a client with specification of scopes.
+
+```sh
+curl -X POST -k -H 'Content-Type: application/json' -i https://localhost:9300/oauth2/default/registration --data '{
+   "application_type": "private",
+   "redirect_uris":
+     ["https://client.example.org/callback"],
+   "post_logout_redirect_uris":
+     ["https://client.example.org/logout/callback"],
+   "client_name": "A Private App",
+   "token_endpoint_auth_method": "client_secret_post",
+   "contacts": ["me@example.org", "them@example.org"],
+   "scope": "openid user/Organization.read"
```

Comment:
Have to include the fhirUser scope or a api:fhir api:pofh scope so server knows these scopes scope.